models/attribute/src/attribute-draft/builder.spec.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TAttributeDraft, TAttributeDraftGraphql } from '../types';
+import * as AttributeDraft from '.';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TAttributeDraft, TAttributeDraft>(
+      'default',
+      AttributeDraft.random(),
+      expect.objectContaining({
+        name: expect.any(String),
+        value: null,
```

Comment:
Should we have a test where we define a value for it?

Reply:
Oh, good call! I never thought of writing a test outside of the boilerplate that we currently use 🧠 💡